pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -56,7 +56,7 @@ const (
 
 	kustomizeDownloadLink  = "https://kubernetes-sigs.github.io/kustomize/installation/"
 	kustomizeMinVersion    = "v3.2.3"
-	kustomizeVersionRegexP = `{Version:(\S+) GitCommit:\S+ BuildDate:\d{4}-\d{2}-\d{2}T\d\d:\d\d:\d\dZ GoOs:\S+ GoArch:\S+}`
+	kustomizeVersionRegexP = `{Version:(\S+) GitCommit:\S+ BuildDate:\d{4}-\d{2}-\d{2}T\d\d:\d\d:\d\d(Z|[+-]\d\d:\d\d) GoOs:\S+ GoArch:\S+}`
```

Comment:
I see the problem. You may not want to add a "group" to the BuildDate tail. The new logic (group > 3) will never be touched. 

For the previous logic, the group count can be either 0 or 2 meaning it either matches nothing (0) or matches exactly (group[0] is the whole string, and group[1] is the Version part). 😸